README.md

```diff
@@ -1,147 +1,202 @@
-Schrijf een [debriefing](Debriefing.md) van de opdracht en check deze met de opdrachtgever en je coach.
+- Users need to be able to chat with each other in voice memos through a chat functionality
+- Users need to be able to take pictures and submit them
+- Users need to be able to provide a spoken explanation with the use of the dictaphone functionality
 
-1. Maandag is kickoff met de opdrachtgever en heb je een standup en checkup met je coach.
-2. Dinsdag stuur je een debriefing.
-3. Woensdag is er ruimte voor workshops en code/design reviews.
-4. Donderdag is er ruimte voor workshops en code/design reviews.
-5. Vrijdag presenteer je je 1e iteratie van het prototype aan de opdrachtgever.
+For the client it is important that she has a clear idea of our progress during this project. To follow the progress of this project it is important to the client that we submit a prototype at the end of every week and discuss the current progress.
 
-### Meesterproef - Week 2 (24 - 28 mei)
+### CMD
 
-De derde week ga je weer testen bij de opdrachtgever. Aan het eind van de week laat je de 2e iteratie van het protoype zien.
+This project has been made available to us by the the course Communication & Multimediadesign of the Hogeschool van Amsterdam. For this project we are assigned a coach (Vasilis van Gemert) that will coach us during the process of this project. Every week we will be meeting with our coach to discuss the current progress of the project and receive feedback and guidance. To be graded at the end of the project we will have to provide the following deliverables:
 
-1. Maandag is het 2e pinksterdag.
-2. Dinsdag heb je een standup en checkup met je coach
-3. Woensdag is er ruimte voor workshops en code/design reviews.
-4. Donderdag is er ruimte voor workshops en code/design reviews.
-5. Vrijdag presenteer je je 2e iteratie van het prototype aan de opdrachtgever en is er een Q&A over de verwachtingen, criteria, leerdoelen en beoordeling.
+- Design Rationale
+- Product Biography
+- Individual reflection on the project
+- A happy customers
 
-### Meesterproef - Week 3 (31 mei - 4 juni)
+Based on these 4 deliverables we will be graded.
 
-De derde week ga je weer testen bij de opdrachtgever. Aan het eind van de week laat je de 3e iteratie van het protoype zien.
+### Target audience
 
-1. Maandag heb je een standup en checkup met je coach
-2. Woensdag is er ruimte voor workshops en code/design reviews.
-3. Donderdag is er ruimte voor workshops en code/design reviews.
-4. Vrijdag presenteer je je 3e iteratie van het prototype aan de opdrachtgever.
+The target audience consists of two groups. The functional illiterate minorities and the volunteers that will help them explaining the letters. It is important to the target audience that an application is provided that fullfills their needs. For the functional illiterate minorities it is important that the application has the following aspects:
```

Comment:
Change "functional illiterate minorites" to "functional illiterate immigrants/people".

Reply:
Changed it to "functional illiterate people"